articles/the-future-of-device-management-ep6.md

```diff
@@ -0,0 +1,33 @@
+# Future of device management
+## EP 6: Whitney Champion: Scaling infrastructure automation with open source tools
```

Comment:
I'd rather not have two colons in one title. Could we start the episode title with the number followed by a period, such as "6." instead of "EP 6:"? If this change has too many cascading effects, like updating titles on podcast platforms, I wouldn't worry about it.

Reply:
What about EP 6 - Whitney Champion: ...etc Just playing devil's advocate on it. We can definitely change it. Better to start this process before we're on episode 100 haha

Reply:
If we're going to include the number, I'd go with "Ep. 6 -" or "6." The simplest option would be to omit the number altogether.